config/field_test.yml

```diff
@@ -28,10 +28,35 @@
 ################################################################################
 experiments:
   # NOTE: Our feed strategy testing experiment must begin with "feed_strategy"
-  feed_strategy_starting_20220801:
+  feed_strategy_starting_20220823:
     # NOTE: Required as we want only want to consider for conversion events that
     #       occurred on or after the given start_date.
     started_at: 2022-08-01
+    ended_at: 2022-08-23
```

Comment:
shouldn't there just be the start time only? because it will be the new current test?

Reply:
Aaah, thought I got all the copy pasta. Thanks!